### PR TITLE
Create che-stacks.addon

### PR DESCRIPTION
**Pull request by JamesDrummond**
Need to a addon for replacing stacks so that users on non-unix os can perform stack replacement.

I don't have minishift setup right now so if someone can test that would be great. Figure I would get things started.

### CODE REVIEW

**Comment by JamesDrummond** [comment]
Reference issue. https://github.com/minishift/minishift-addons/issues/60

**Comment by JamesDrummond** [comment]
@budhrg OK made some changes. Let me know if they work. Also, I have been told that the stacks will eventually not have to be replaced to get default openshift stacks included with Che. However, this will still be a good addon for those that may want to use their own stacks. They would just need to change the url in the addon file to point to a different stacks json file url.

File: add-ons/che-stacks/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Eclipse Che Stacks Addon
+
+This is part of the Che Minishift addon. Please refer to README.md in che addon folder.
```
Comment (gbraad) [inline add-ons/che-stacks/README.md]: And this is why we should think of a dependency option to allow related add-ons to be selected/enabled/installed.

File: add-ons/che/README.md [hunk 2]
```diff
@@ -64,10 +65,7 @@ If Minishift is already started and che addon is installed. It is possible to de
 ### Replace Che stacks
 Once Che is up and running,
 
-    $ STACKS_SCRIPT_URL=https://raw.githubusercontent.com/eclipse/che/master/dockerfiles/init/modules/openshift/files/scripts/replace_stacks.sh
-    $ curl -fsSL ${STACKS_SCRIPT_URL} -o ./stacks-che.sh
-    $ oc project mini-che && bash ./stacks-che.sh
-
+    $ minishift addons apply che-stacks
```
Comment (gbraad) [inline add-ons/che/README.md]: 👍

**Comment by JamesDrummond** [comment]
@budhrg Looks like in Che6 this script will no longer be needed for the default stacks so I am going to close the PR.

**Pull request closed without merging**